plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -50,7 +50,9 @@ fu_plugin_thunderbolt_safe_kernel (FuPlugin *plugin, GError **error)
 		return TRUE;
 	}
 
-	if (fu_common_vercmp (name_tmp.release, minimum_kernel) < 0) {
+	if (fu_common_vercmp_full (name_tmp.release,
+				   minimum_kernel,
+				   FWUPD_VERSION_FORMAT_TRIPLET) < 0) {
```

Comment:
I wonder if we need to count the number of dots now in the config value to make sure this stayed triplet.  Otherwise `thunderbolt.conf` having a bad value could be problematic?

Reply:
It's not important really what the verfmt actually is. It's just non-`PLAIN`.